Plugins/PluginWebParser/WebParser.cpp

```diff
@@ -1511,28 +1512,12 @@ BYTE* DownloadUrl(HINTERNET handle, std::wstring& url, DWORD* dataSize, bool for
 			return nullptr;
 		}
 
-		FILE* file = _wfopen(path, L"rb");
-		if (!file)
-		{
-			return nullptr;
-		}
-
-		fseek(file, 0, SEEK_END);
-		*dataSize = ftell(file);
-		rewind(file);
-
-		BYTE* buffer = (BYTE*)malloc(*dataSize + 3);
-		if (buffer)
-		{
-			fread(buffer, 1, *dataSize, file);
-
-			// Triple null terminate the buffer.
-			buffer[*dataSize] = 0;
-			buffer[*dataSize + 1] = 0;
-			buffer[*dataSize + 2] = 0;
-		}
+		// ReadFullFile returns unique_ptr. Raw BYTE* pointer is expected down the line

```

Comment:
@tomkort This comment is true, but obvious from the context. Could you remote it please?